pkg/standalone/stop.go

```diff
@@ -34,8 +34,9 @@ func Stop(appID string) error {
 			var pid string
 			// Kill the Daprd process if Daprd was started without CLI, otherwise
 			// kill the CLI process which also kills the associated Daprd process.
-			if a.CliPID == 0 {
+			if a.CliPID == 0 || cliPIDToNoOfApps[a.CliPID] > 1 {
 				pid = fmt.Sprintf("%v", a.DaprdPID)
+				cliPIDToNoOfApps[a.CliPID]--
```

Comment:
this line is not needed right? 
As the command is run only once and then the `dapr stop` command exits with the return statement below.

Reply:
it can be triggered like this -  dapr stop app1 app2 app3 (all 3 apps are started by dapr run -f )